test/functional/wallet_migration.py

```diff
@@ -485,11 +485,16 @@ def test_encrypted(self):
 
         # Use self.migrate_and_get_rpc to test this error to get everything copied over to the master node
         assert_raises_rpc_error(-4, "Error: Wallet decryption failed, the wallet passphrase was not provided or was incorrect", self.migrate_and_get_rpc, "encrypted")
+
         # Use the RPC directly on the master node for the rest of these checks
+        self.master_node.bumpmocktime(1) # Prevents filename duplication on wallet backups as they contain the time in the filename
```

Comment:
In c78f183800db663bba5f9d3a7280583127eebe3f: It would be good for the commit messages and/or the added doc, to explain that this is (I'm assuming?) only an issue on Windows. If it isn't, and it effects all other platforms, then these test changes could be split out, with a reproducer showing the issue.

Reply:
It was only a problem on Windows. Made a note of that.